docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[cases-open-manage]]
+= Open and manage cases
+
+You can create and manage cases using the UI or the <<cases-api-overview>>.
+
+[float]
+[[cases-ui-open]]
+== Open a new case
+
+Open a new case to keep track of security issues and share their details with
+colleagues.
+
+. Go to *Investigate* -> *Cases*, then click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table.
+. Give the case a name, add relevant tags, assign a severity level, and provide a description.
++
+NOTE: If you do not assign your case a severity level, it will be assigned *Low* by default.
+
++
+TIP: In the case description, you can use
+https://www.markdownguide.org/cheat-sheet[Markdown] syntax and insert a Timeline link (click the image:images/add-timeline-button.png[Click Timeline icon,17,17] button).
+
+. Choose whether you want alert statuses to sync with the case's status after they are added to the case. This option is enabled by default, but you can still turn it off after creating the case.
+. From *External incident management*, select a <<cases-ui-integrations, connector>>. If you’ve previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
+. Click *Create case*.
+
+
+[role="screenshot"]
+image::images/cases-ui-open.png[Shows an open case]
+
+[float]
+[[cases-ui-manage]]
+== Manage existing cases
+
+From the Cases page, you can search existing cases and filter them by severity, tags, reporter, and status: *open*, *in-progress*, or *closed*. General case metrics, including how long it takes to close cases, are provided above the table.
```

Comment:
To match UI. Also I don't think bold is needed here bc we're not telling the user to interact with them:
```suggestion
From the Cases page, you can search existing cases and filter them by severity, tags, reporter, and status: Open, In progress, or Closed. General case metrics, including how long it takes to close cases, are provided above the table.
```